.github/workflows/build_test.yml

```
@@ -254,7 +254,7 @@ jobs:
       - name: Install tox
         run: python -m pip install tox tox-external-wheels
       - name: Run tests
-        run: python -m tox -e "py${PYTHON_VERSION/\./}-{extras,unit,integ,pandas,sso}-ci"
+        run: python -m tox -e "py${PYTHON_VERSION/\./}-{extras,unit,integ,pandas,sso}-ci-parallel"
```

Let's be more specific, with this change only `unit` has a `parallel` option.
```suggestion
        run: python -m tox -e "py${PYTHON_VERSION/\./}-{extras,unit-parallel,integ,pandas,sso}-ci"
```